README.md

```diff
@@ -95,6 +95,13 @@ Other useful commands:
         kob version     
          
 
+### Manual Release
```

Comment:
Move instructions to  document maintainers-handbook.md

Have  section preparing for Release.

Reply:
maintainers-handbook.md
     only the instructions for manual release??
Have section preparing for Release.
     Please do guide me on this

Reply:
Have a document for Maintainers , doing a release will be on activity for them .

Move this under that.